Yoast/wordpress-seo
# Use the helper instead of the util.

Also deprecated the util.

## Context


* We have to use helpers instead of the good old static utils.

## Summary


This PR can be summarized in the following changelog entry:

* Use that already present helper method to check if the WooCommerce plugin is active.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* There is no functional change:
* Check if the WooCommerce plugin is visible on the 'premium' page when having WooCommerce active
* Check if the settings for WooCommerce products under search appearence say something about the shop page settings.

## UI changes

## Documentation

## Quality assurance


Fixes #
